### DISCUSSION TITLE
반복 주기의 dateOfYear 필드에 대한 논의 사항

### DISCUSSION THREAD

**Discussion by KIMSEI1124** [What-s-Your-Plan/wypl-backend#13] [General]
# 개요

- #12  해당 PR에서 논의 사항이 필요한 주제가 나오게 되었습니다.

`Repetition.java` 파일에 아래와 같은 변수가 있는데 해당 변수에 대한 논의가 필요합니다.

```java
@Column(name = "date_of_year")
private Object dateOfYear;      // 논의 필요
```

자세한 이야기는 회의를 진행하면서 기록하도록 하겠습니다.

**Comment by jiwon83** [comment]
년도 기준 반복일정 구현 진행하겠습니다.
